js/WebAPI/DocumentEditor.md

```diff
@@ -1,20 +1,20 @@
 ---
 layout: post
-title: webAPI reference for DocumentEditor
-description: webAPI reference for 
+title: WebAPI reference in JavaScript DocumentEditor Control | Syncfusion
+description: Learn here all about webAPI reference support in Syncfusion JavaScript DocumentEditor control, its elements and more.
 documentation: ug
 platform: js
 keywords: documenteditor,syncfusion,documenteditor webapi
 api: /api/js/ejdocumenteditor
 ---
 
-## Import
+# Import in JavaScript DocumentEditor Control
```

Comment:
Update like this - WebAPI Reference in JavaScript Document Editor Control

Reply:
Resolved

Reply:
resolved